students/Zach Cooper/lesson01/inventory_management/main.py

```diff
@@ -0,0 +1,126 @@
+"""
+This file contains all the code necessary to run the main menu
+"""
+
+
+import sys
+from inventory_management.market_prices import get_latest_price
+from inventory_management.inventoryClass import Inventory
+from inventory_management.furnitureClass import Furniture
+from inventory_management.electricAppliancesClass import ElectricAppliances
+# import sys
+# import market_prices
+# import inventoryClass
+# import furnitureClass
+# import electricAppliancesClass
```

Comment:
Main is in the same directory as market_prices, inventoryClass, etc. so the commented out imports are really the ones you want. If main was in the directory above then you would need to import them from their directory. Also, you are using them down below as market_prices.get_latest_price, etc, so those imports would not work even if this was in the directory above.